keras/backend/mxnet_backend.py

```diff
@@ -4182,13 +4213,140 @@ def _preprocess_padding_mode(padding_mode, input_shape, kernel, strides, dilatio
 
 
 def _layout_kernel(kernel):
-
     layout_kernel = tuple(kernel[2:])
     nb_filter = kernel[0]
 
     return layout_kernel, nb_filter
 
 
+def _mx_dw_conv(data, num_in_channel, weight=None, kernel=(3, 3), stride=(1, 1), pad=(1, 1), name=None,
+                depth_mult=1):
+    # pure mxnet version of Separable Convolution
+    # depthwise convolution
+    channels = mx.sym.split(data=data, axis=1, num_outputs=num_in_channel)  # for new version of mxnet > 0.8
+
+    depthwise_outs = [
```

Comment:
Should we add checks to make sure Stride is (1,1)?

Reply:
updated to always use stride (1,1)